src/vcf-pdf-viewer.js

```diff
@@ -550,6 +536,19 @@ class PdfViewerElement extends
         this.__resizeObserver = new ResizeObserver(() => {
             requestAnimationFrame(() => this.__recalculateSizes());
         });
+        this.$.zoom.items = [
```

Comment:
The first two item labels are only set once on `ready` with the initial values of `this.autoZoomOptionLabel` and `this.fitZoomOptionLabel`, which means changing these properties later would not update the select options. I'd recommend the following approach:

Add a computed property for the zoom items:

```js
__zoomItems: {
  computed: '__computeZoomItems(autoZoomOptionLabel, fitZoomOptionLabel)'
}

```

Then define the compute function:

```js
__computeZoomItems(autoZoomOptionLabel, fitZoomOptionLabel) {
  return [
    { label: autoZoomOptionLabel, value:'auto' },
    { label: fitZoomOptionLabel, value:'page-fit' },
    { label: '50%', value:'0.5' },
    { label: '75%', value:'0.75' },
    { label: '100%', value:'1.0' },
    { label: '125%', value:'1.25' },
    { label: '150%', value:'1.5' },
    { label: '200%', value:'2.0' },
    { label: '300%', value:'3.0' },
    { label: '400%', value:'4.0' }
 ]
}
```

Finally, bind the new property to the field:
```js
<vaadin-select id="zoom" part="toolbar-zoom" value="{{zoom}}" items="[[__zoomItems]]">
```

Alternatively, you can inline the compute function directly and skip adding the new property:
```js
<vaadin-select id="zoom" part="toolbar-zoom" value="{{zoom}}" items="[[__computeZoomItems(autoZoomOptionLabel, fitZoomOptionLabel)]]">
```